test/utilities/DominoDocumentFragment.test.js

```diff
@@ -0,0 +1,29 @@
+import assert from 'assert'
+import domino from 'domino'
+import dominoDocumentFragment from './DominoDocumentFragment'
+
+const documentFragmentFromHTMLString = dominoDocumentFragment.documentFragmentFromHTMLString
+
+describe('DominoDocumentFragment', () => {
+  describe('documentFragmentFromHTMLString()', () => {
+    it('should return a Domino DocumentFragment when given an HTML string', () => {
+      const frag =
+        documentFragmentFromHTMLString('<div>Oh look! A fragment!</div><a id="link1"></a>')
+      assert.ok(frag instanceof domino.impl.DocumentFragment)
+    })
+    it('should return a Domino DocumentFragment with expected childNodes', () => {
+      const frag =
+        documentFragmentFromHTMLString('<div>Oh look! A fragment!</div><a id="link1"></a>')
+      assert.ok(frag.childNodes.length === 2)
+      const firstChild = frag.childNodes[0]
```

Comment:
Mega style nerd: isolating `frag.childNodes[0]` seems obvious enough as it is that it's the first child and it only has one usage. Consider dropping `firstChild` and `secondChild` for readability.